src/drawer/src/components/styled/DrawerCard.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import styled, { css } from 'styled-components';
+import { WHITE } from '../../../../colors';
+
+type Props = {
+  side :'left' | 'right';
+};
+
+const getPosition = (props :Props) => {
+  const { side } = props;
+  switch (side) {
+    case 'right':
+      return css`
+        right: 0;
+      `;
+
+    case 'left':
+    default:
+      return css`
+        left: 0;
+      `;
+  }
+};
+
+const DrawerCard = styled.div`
+  position: fixed;
+  background-color: ${WHITE};
+  height: 100%;
+  width: 300px;
```

Comment:
A fixed width is fine for now, but I'd imagine we'll want this to be more flexible in the future.